module/CameraHandler.py

```diff
@@ -0,0 +1,41 @@
+# includes for the python BUS
+from client.comm import BaseComm
+from common.frame_enum import FrameType
+
+
+class CameraHandler:
+    def __init__(self, comm: BaseComm):
+        self.comm = comm
+        self.comm.listen_for([FrameType.PLACEHOLDER])  # Implement frametype ASAP!
+        """ Should detect which camera's are present"""
+        pass
+
+    def capture(self):
+        """ Instructs a camera to take an image"""
+        pass
+
+    def process(self):
+
+        while self.comm.has_data():
+            frame = self.comm.get_data()
+
+            if frame.request:
+                continue
+
+            values = frame.get_data()
+
+            if values[0] == "record":
+                self.record(values(1))  # values[1] should hold the time
+            else:
+                continue  # should add more functionality later?
```

Comment:
Indeed. I will comment it out for now. It was made due to the possibility of multiple frame types. I'm afraid I will not be picking that up for now so it will be omitted with proper documentation.

Reply:
Thought about for a minute and decided to change my mind. I've removed it entirely. It's up to the future implementation to decide it and the current code is vague and outdated.